docs/external-node.md

```diff
@@ -0,0 +1,305 @@
+# ExternalNode
+
+## Table of Contents
+
+<!-- toc -->
+- [What is ExternalNode?](#what-is-externalnode)
+- [Prerequisites](#prerequisites)
+- [The ExternalNode resource](#the-externalnode-resource)
+  - [Name and namespace](#name-and-namespace)
+  - [Interfaces](#interfaces)
+- [Apply Antrea NetworkPolicy to ExternalNode](#apply-antrea-networkpolicy-to-externalnode)
+- [Security](#security)
+- [Traffic model](#traffic-model)
+  - [OVS pair ports](#ovs-pair-ports)
+  - [OpenFlow pipeline](#openflow-pipeline)
+    - [Non-IP packet](#non-ip-packet)
+    - [IP packet](#ip-packet)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## What is ExternalNode?
+
+`ExternalNode` is a CRD API that manages the network connectivity and security on Non Kubernetes Node (like a virtual
```

Comment:
that enables Antrea to manage the network connectivity...

Reply:
done.